conans/client/generators/cmake_find_package_multi.py

```diff
@@ -0,0 +1,102 @@
+    
+# Load the debug and release library finders
+get_filename_component(_DIR "${{CMAKE_CURRENT_LIST_FILE}}" PATH)
+file(GLOB CONFIG_FILES "${{_DIR}}/Find{name}-*.cmake")
+
+if(NOT ${{CMAKE_VERSION}} VERSION_LESS "3.0")
+    # Target approach
+    if(NOT TARGET {name}::{name})
+        add_library({name}::{name} INTERFACE IMPORTED)
+    endif()
+endif()
+
+foreach(f ${{CONFIG_FILES}})
+  include(${{f}})
+endforeach()
+
+include(SelectLibraryConfigurations)
+select_library_configurations({name})
+
+# CMake doesn't have a mechanism to select also the right include directories, assume we can use any of them or use
+# the variable _DEBUG or _RELEASE
+set({name}_INCLUDE_DIRS ${{{name}_INCLUDE_DIRS_RELEASE}} ${{{name}_INCLUDE_DIRS_DEBUG}})
+
+if(NOT ${{CMAKE_VERSION}} VERSION_LESS "3.0")
+    {target_props_block}
+    {find_dependencies_block}
+endif()
+
+"""
+
+    target_properties = """
+    # Assign target properties
+    set_property(TARGET {name}::{name} 
+                 PROPERTY INTERFACE_LINK_LIBRARIES 
+                     $<$<CONFIG:Release>:${{{name}_LIBRARIES_TARGETS_RELEASE}} ${{{name}_LINKER_FLAGS_RELEASE_LIST}}>
+                     $<$<CONFIG:RelWithDebInfo>:${{{name}_LIBRARIES_TARGETS_RELEASE}} ${{{name}_LINKER_FLAGS_RELEASE_LIST}}>
+                     $<$<CONFIG:MinSizeRel>:${{{name}_LIBRARIES_TARGETS_RELEASE}} ${{{name}_LINKER_FLAGS_RELEASE_LIST}}>
+                     $<$<CONFIG:Debug>:${{{name}_LIBRARIES_TARGETS_DEBUG}} ${{{name}_LINKER_FLAGS_DEBUG_LIST}}>)
+    set_property(TARGET {name}::{name} 
+                 PROPERTY INTERFACE_INCLUDE_DIRECTORIES 
+                     $<$<CONFIG:Release>:${{{name}_INCLUDE_DIRS_RELEASE}}>
+                     $<$<CONFIG:RelWithDebInfo>:${{{name}_INCLUDE_DIRS_RELEASE}}>
+                     $<$<CONFIG:MinSizeRel>:${{{name}_INCLUDE_DIRS_RELEASE}}>
+                     $<$<CONFIG:Debug>:${{{name}_INCLUDE_DIRS_DEBUG}}>)
+    set_property(TARGET {name}::{name} 
+                 PROPERTY INTERFACE_COMPILE_DEFINITIONS 
+                     $<$<CONFIG:Release>:${{{name}_COMPILE_DEFINITIONS_RELEASE}}>
+                     $<$<CONFIG:RelWithDebInfo>:${{{name}_COMPILE_DEFINITIONS_RELEASE}}>
+                     $<$<CONFIG:MinSizeRel>:${{{name}_COMPILE_DEFINITIONS_RELEASE}}>
+                     $<$<CONFIG:Debug>:${{{name}_COMPILE_DEFINITIONS_DEBUG}}>)
+    set_property(TARGET {name}::{name} 
+                 PROPERTY INTERFACE_COMPILE_OPTIONS 
+                     $<$<CONFIG:Release>:${{{name}_COMPILE_OPTIONS_RELEASE_LIST}}>
+                     $<$<CONFIG:RelWithDebInfo>:${{{name}_COMPILE_OPTIONS_RELEASE_LIST}}>
+                     $<$<CONFIG:MinSizeRel>:${{{name}_COMPILE_OPTIONS_RELEASE_LIST}}>
+                     $<$<CONFIG:Debug>:${{{name}_COMPILE_OPTIONS_DEBUG_LIST}}>) 
+    """
+
+    @property
+    def filename(self):
+        pass
+
+    @property
+    def content(self):
+        ret = {}
+        build_type = self.conanfile.settings.get_safe("build_type")
```

Comment:
What will happen for users with ``-s build_type=RelWithDebInfo``? Shouldn't they be mapped to ``Release`` the same way the targets properties are?